src/transformers/models/phi/configuration_phi.py

```diff
@@ -148,6 +167,7 @@ def __init__(
         self.rope_scaling = rope_scaling
         self.partial_rotary_factor = partial_rotary_factor
         self.qk_layernorm = qk_layernorm
+        self.pad_vocab_size_multiple = pad_vocab_size_multiple
```

Comment:
that's not something we should need / want tbh! The doc should explain that the vocab was padded!